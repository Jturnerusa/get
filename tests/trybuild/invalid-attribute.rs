use get::Get;

#[derive(Get)]
pub struct Cat {
    #[get()]
    name: String,
}

#[derive(Get)]
pub struct Dog {
    #[get]
    name: String,
}

#[derive(Get)]
pub struct Crab {
    #[get(invalid_name = "invalid_value")]
    name: String,
}

pub fn main() {}
