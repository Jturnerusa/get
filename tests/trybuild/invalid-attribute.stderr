error: proc-macro derive panicked
 --> tests/trybuild/invalid-attribute.rs:3:10
  |
3 | #[derive(Get)]
  |          ^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: "failed to parse attribute"

error: proc-macro derive panicked
 --> tests/trybuild/invalid-attribute.rs:9:10
  |
9 | #[derive(Get)]
  |          ^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: "failed to parse attribute"

error: proc-macro derive panicked
  --> tests/trybuild/invalid-attribute.rs:15:10
   |
15 | #[derive(Get)]
   |          ^^^
   |
   = help: message: called `Result::unwrap()` on an `Err` value: "invalid name value list in attribute"
